_includes/kubernetes-mac-win.md

```diff
@@ -0,0 +1,176 @@
+{% assign platform = include.platform %}
+
+{% comment %}
+
+Include a chunk of this file, using variables already set in the file
+where you want to reuse the chunk.
+
+Usage: {% include kubernetes-mac-win.md platform="mac" %}
+
+{% endcomment %}
+
+{% if platform == "mac" %}
+  {% assign product = "Docker for Mac" %}
+
+  {% capture min-version %}{{ product }} 17.12 CE Edge{% endcapture %}
+
+  {% capture version-caveat %}
+**Kubernetes is only available in Docker for Mac 17.12 CE Edge.** Kubernetes
+support is not included in Docker for Mac 17.12 CE Stable.
+  {% endcapture %}
+
+  {% capture local-kubectl-warning %}
+> If you independently installed the Kubernetes CLI, `kubectl`, make sure that
+> it is pointing to `docker-for-desktop` and not some other context such as
+> `minikube` or a GKE cluster. Run: `kubectl config use-context docker-for-desktop`.
+> If you experience conflicts with an existing `kubectl` installation, remove `/usr/local/bin/kubectl`.
+
+  {% endcapture %}
+
+  {% assign kubectl-path = "/usr/local/bin/kubectl" %}
+
+{% elsif platform == "windows" %}
+  {% assign product = "Docker for Windows" %}
+
+  {% capture min-version %}{{ product }} 18.02 CE Edge{% endcapture %}
+
+  {% capture version-caveat %}
+**Kubernetes is only available in Docker for Windows 18.02 CE Edge users who
+participants in the [Docker Beta program](https://beta.docker.com/). To access
```

Comment:
"Kubernetes is only available in Docker for Windows 18.02 CE Edge. ", we don't do a private beta for the windows kube integration (since we already got feedback from the mac users. No need to log in.